longevity_alternator_ttl_test.py

```diff
@@ -0,0 +1,173 @@
+import time
+
+from longevity_test import LongevityTest
+from sdcm.cluster import BaseNode
+from sdcm.utils.alternator.consts import NO_LWT_TABLE_NAME
+
+
+class AlternatorTtlLongevityTest(LongevityTest):
+
+    keyspace = f"alternator_{NO_LWT_TABLE_NAME}"
+    full_table_name = f'{keyspace}.{NO_LWT_TABLE_NAME}'
+
+    def _count_sstables_and_partitions(self, node: BaseNode) -> int:
+        estimated_num_of_partitions = node.get_cfstats(self.full_table_name)['Number of partitions (estimate)']
+        num_of_sstables = node.get_cfstats(self.full_table_name)['SSTable count']
```

Comment:
```suggestion
        cfstats = node.get_cfstats(self.full_table_name)
        estimated_num_of_partitions =cfstats['Number of partitions (estimate)']
        num_of_sstables = cfstats['SSTable count']
```

AFAIK `cfstats` is a heavy command, so you don't need to run it twice in a row just to get 2 different information from it, but you can run it once, save its result in a new variable and re-use that variable, twice

Reply:
fixed